sci-physics/xfoil/xfoil-6.97-r1.ebuild

```diff
@@ -33,39 +35,39 @@ src_prepare() {
 		|| die "sed for flags and compilers failed"
 
 	# fix bug #147033
-	[[ $(tc-getFC) == *gfortran ]] && \
-		epatch "${FILESDIR}"/${PN}-6.96-gfortran.patch
+	[[ $(tc-getFC) == *gfortran ]] && PATCHES+=( "${FILESDIR}"/${PN}-6.96-gfortran.patch )
 
-	epatch "${FILESDIR}"/${P}-overflow.patch
+	default
```

Comment:
if possible, always try to do patching (i.e. `default`) at the top of a `src_prepare`. You would have to move the gfortran test in front too.